tools/chromiumos/gen_autosuspend_rules.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python3
-# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
What's this change for?

Reply:
Not sure, the command `meson compile -C build update-hwdb update-hwdb-autosuspend` removes it.

Reply:
The .py file is downloaded from https://chromium.googlesource.com/chromiumos/platform2/+/master/power_manager/udev/gen_autosuspend_rules.py?format=TEXT

Reply:
Please exclude this change from the commit.

Reply:
Hm, we may need to adjust `update-hwdb-autosuspend.sh`. In this time. I will just drop this and below changes.

Reply:
> Please exclude this change from the commit.

Sure. Done.

Reply:
It'd be nicer to submit *our* change as a patch for the upstream project.